src/generator.d

```diff
@@ -743,7 +743,19 @@ string[] generate(Module mod)
     string[] externs;
     foreach (ext; state.externs)
     {
-        externs ~= format("extern %s", ext);
```

Comment:
They get added to the extern list here:

https://github.com/eatonphil/bshift/blob/476e2fec030c483b0d2b436002e19cd169b5876f/src/generator.d#L2495

Which looks like it should work right now. It's checking if the FunctionSignature.mod (module) is the current module being compiled. Wonder why that check doesn't work now.